execution/src/evm.rs

```diff
@@ -174,19 +159,19 @@ impl<'a, R: ExecutionRpc> Evm<'a, R> {
         let mut env = Env::default();
         let payload = &self.evm.db.as_ref().unwrap().current_payload;
 
-        env.tx.transact_to = TransactTo::Call(opts.to.unwrap_or_default());
-        env.tx.caller = opts.from.unwrap_or(Address::zero());
-        env.tx.value = opts.value.unwrap_or(U256::from(0));
+        env.tx.transact_to = TransactTo::Call(opts.to.unwrap_or_default().into());
+        env.tx.caller = opts.from.unwrap_or(Address::zero()).into();
+        env.tx.value = opts.value.unwrap_or(U256::from(0)).into();
         env.tx.data = Bytes::from(opts.data.clone().unwrap_or_default());
         env.tx.gas_limit = opts.gas.map(|v| v.as_u64()).unwrap_or(u64::MAX);
-        env.tx.gas_price = opts.gas_price.unwrap_or(U256::zero());
+        env.tx.gas_price = opts.gas_price.unwrap_or(U256::zero()).into();
 
-        env.block.number = U256::from(*payload.block_number());
-        env.block.coinbase = Address::from_slice(payload.fee_recipient());
-        env.block.timestamp = U256::from(*payload.timestamp());
-        env.block.difficulty = U256::from_little_endian(payload.prev_randao());
+        env.block.number = U256::from(*payload.block_number()).into();
+        env.block.coinbase = Address::from_slice(payload.fee_recipient()).into();
+        env.block.timestamp = U256::from(*payload.timestamp()).into();
+        env.block.difficulty = U256::from_little_endian(payload.prev_randao()).into();
 
-        env.cfg.chain_id = self.chain_id.into();
+        env.cfg.chain_id = self.chain_id.try_into().unwrap();
```

Comment:
is unwrap safe there?

Reply:
yes. it converts a u64 to u256. i can use expect is prefer

Reply:
added `expect`